security/fma-l1-pectra.md

```diff
@@ -0,0 +1,96 @@
+- **Risk Assessment:** High severity, Medium Likelihood
+- **Mitigations:**
+  - The superchain-ops tasks will include both manual checking (in Validations.md) and automated checking (in NestedSignFromJson.s.sol). Thus although the manner of executing the upgrade is changing, we are maintaining the
+    pre-existing methods of fully validating the resulting system.
+- **Detection:**
+  - If a system is misconfigured despite the mitigations, we would not have a predictable method of detecting such issues.
+- **Recovery Path(s)**: Any bugs would need to be patched as soon as possible. We can fallback to the old upgrade path using superchain-ops tasks with manually prepared calldata.
+
+## Generic Items
+
+Although this upgrade is technically a soft fork (it does not need to be coordinated across nodes other than being applied before Pectra activates on L1) many of the items in [./fma-generic-hardfork.md](./fma-generic-hardfork.md) apply. In particular:
+
+- Chain Halt at activation
+- Invalid `DisputeGameFactory.setImplementation` execution.
+- Chain split across clients
+
+- [x] Check this box to confirm that these items have been considered and updated if necessary.
+
+## Audit Requirements
+
+## Action Items
+
+Below is what needs to be done before launch to reduce the chances of the above failure modes occurring, and to ensure they can be detected and recovered from:
+
+- [ ] (BLOCKING): Resolve all comments on this document and incorporate them into the document itself (Assignee: document author)
+- [x] (BLOCKING): Action tests will be added which are run on op-node and Kona https://github.com/ethereum-optimism/optimism/issues/13967. Ideally they run against the usual mainline version of op-geth -- if this is not possible the tests can run against a patched version until op-geth is ready to support such tests.
+- [x] (BLOCKING): The changes will be deployed to a local multi-client kurtosis devnet with both op-geth and op-reth running as well as Pectra activated on L1. https://github.com/ethereum-optimism/optimism/pull/14046
+- [x] (BLOCKING): we will modify the batcher to cope with the change in calldata which activates with Pectra https://github.com/ethereum-optimism/optimism/releases/tag/op-batcher%2Fv1.11.0 so that it can continue to switch DA mode (between blobs and calldata) appropriately.
+- [ ] (non-BLOCKING) The changes will be deployed to a devnet which targets a public, Pectra-enabled L1 devnet.
+- [x] (non-BLOCKING): We will update the op-sepolia and op-mainnet vm-runners to use the new absolute prestate. The vm-runner runs the op-program in the MIPS FPVM using inputs sampled from a live chain.
+
+Additional action items are copied here from the [generic hardfork FMA](./fma-generic-hardfork.md) doc:
+
+- [ ] (non-BLOCKING): The superchain-ops task to upgrade any contract should check if the semantic versions and bytecodes after the upgrade are as expected.
```

Comment:
we should add this AI: 
- [ ]  Having tests on the OptimismPortal2 force inclusion with different type (for safeCodeTx).